src/Resources/config/phpcr.xml

```diff
@@ -48,7 +48,7 @@
         <service id="doctrine_phpcr.data_collector"
                  class="Doctrine\Bundle\PHPCRBundle\DataCollector\PHPCRDataCollector"
                  public="false">
-            <tag name="data_collector" template="DoctrinePHPCRBundle:Collector:phpcr" id="phpcr" priority="247" />
+            <tag name="data_collector" template="DoctrinePHPCR/Collector/phpcr" id="phpcr" priority="247" />
```

Comment:
It think it should be `@DoctrinePHPCR/Collector/phpcr.html.twig`.

Does it work for you with this syntax ?

Reply:
The `.html.twig` file extension is not required by the Web Profiler.
See https://github.com/symfony/web-profiler-bundle/blob/2074348dba4b49ea86d7ee7eaf1b4fcacc887120/Profiler/TemplateManager.php#L85

But I think the **@** is necessary.

I'm improving my tests in order to show the exceptions while doing some request using the test client. Actually, I just get HTTP error codes. The exception is caught by the Twig error controller.

Reply:
You're right, the `@` is missing. Sorry about that, I'm doing a new PR right now.

Reply:
Not your fault. There isn't any tests. #313 will add them.